src/components/interaction-designer/block-editors/ContactPropertyEditor.vue

```diff
@@ -80,6 +81,8 @@ const EMPTY_STRING_EXPRESSION = ''
 @Component({
   components: {
     ExpressionInput,
+    ValidationMessage,
```

Comment:
@safydy-r These were removed as part of testing the changes you made in VMO-442. Components are now registered globally in `main.ts` and `lib.ts`. The effect of this will be to break component overriding I think - e.g. we will always used these local dependencies over what is registered with `Vue.component` - https://github.com/FLOIP/flow-builder/pull/208/files#diff-e59662795ab51f95e1bf2a44d30dd0debbc194a499adffd1bb6a197bd188d4b5R107

Really should have remembered this crucial part when reviewing VMO-442. What we need to do instead to bring these dependencies back in is something like the loop in that link in `/story-utils/storeSetup` or somewhere similarly global to all stories.

We also need to remove all `  components: {` everywhere. This should have happened in VMO-442 - was a crucial part and something that QA could have actually caught! Would have beem quite and awkward QA though as we would have to compare a branch where some things are overridden with one where they aren't

Reply:
Thanks @jcbashdown for this input. As agreed in slack, feel free to implement your suggestion and let us know when you have a ready PR ;) 
Thanks in advance